apps/nextra/pages/en/build/indexer/custom-processors.mdx

```diff
@@ -8,4 +8,17 @@ import { IndexerBetaNotice } from '@components/index';
 
 <IndexerBetaNotice />
 
-This section explains what a custom processor is, how to write and deploy one, and how to parse transactions from the [Transaction Stream Service](txn-stream.mdx).
+Everything the Indexer API exposes is initially tracked through writing a processor to read in the transaction data (or other off-chain service) then use it to update the Indexer's database.
+
+These processors are often used to track:
+1. Historical data (by continuously watching and updating as things change).
+2. Aggregate data (by collecting data across the chain).
```

Comment:
Is this basically "current data"?

Reply:
I wouldn't say so - I was trying to get at the idea that you can look up information across the chain as a whole instead of one transaction at a time. (So less about the timing of the data, and moreso being able to handle lots of it all at once with queries :) )